.github/ISSUE_TEMPLATE/issue--client-catapult.md

```diff
@@ -0,0 +1,41 @@
+---
+name: '[client/catapult] Bug Report'
+about: Got a bug with the client? Fill out this form!
+title: "[client/catapult] Bug Report"
```

Comment:
Since it was mentioned in #47 - " " is mandatory for the `title field` in issue reports. ' ' is used for the `name field`.